docroot/themes/custom/twentyseventeen/sass/components/schedule/_schedule.scss

```diff
@@ -0,0 +1,134 @@
+	//border: 1px solid #f00;
+	
+}
+
+
+.program-schedule .views-exposed-form .form-item.js-form-type-select.form-type-select.form-item-sort-by,
+.program-schedule .views-exposed-form .form-item.js-form-type-select.form-type-select.form-item-sort-order,
+.program-schedule .views-exposed-form .form-item.js-form-type-select.form-type-select.form-item-field-experience-level-value,
+.program-schedule .views-exposed-form .form-item.js-form-type-select.form-type-select.form-item-field-session-track-target-id,
+.program-schedule .views-exposed-form .form-actions.js-form-wrapper.form-wrapper {
+	display: inline-block;
+}
+
+.program-schedule .views-exposed-form .form-actions.js-form-wrapper.form-wrapper,
+.program-schedule .views-exposed-form {
+	//margin-bottom: 1em;
+	//border: 1px solid #f00;
+	margin-bottom: 0;
+}
+
+.program-schedule #edit-field-schedule-date-value,
+.program-schedule #edit-field-schedule-time-value {
+	//border: 1px solid #0f0;
+	padding-left: 0.27em;
+	
+}
+
+.program-schedule .views-exposed-form select {
+	width: 8em;
+}
+
+
+.program-schedule #edit-field-schedule-date-value > div:first-child,
+.program-schedule #edit-field-schedule-time-value > div:first-child {
+	display: none;
+}
+
+.program-schedule #edit-field-schedule-date-value > div,
+.program-schedule #edit-field-schedule-time-value > div {
+	display: inline-block;
+	font-weight: 400;
+	font-size: 1.2em;
+}
+
+.program-schedule #edit-field-schedule-date-value > div a:link,
+.program-schedule #edit-field-schedule-date-value > div a:visited,
+.program-schedule #edit-field-schedule-time-value > div a:link,
+.program-schedule #edit-field-schedule-time-value > div a:visited {
+	padding: 0.5em;
+	color: rgba(255,255,255,1);
+	background: rgba($green,0.75);
+	text-decoration: none;
+	margin-left: -0.23em;
+}
+
+.program-schedule #edit-field-schedule-date-value > div a.bef-link-active,
+.program-schedule #edit-field-schedule-time-value > div a.bef-link-active {
+	background: rgba($green,1);
+}
+
+
+.program-schedule .view-grouping-header {
+	text-transform: uppercase;
+	font-size: 1.5em;
+	font-weight: bold;
+	color: $wht;
+	letter-spacing: 0.02em;
+	background: rgba($green,1);
+	border-top: 2px solid $green;
+	padding: 0.5em 0.5em 0.375em 0.5em;
+}
+
+.program-schedule .view-grouping {
+	margin-top: -1em;
+}
+
+.program-schedule .view-grouping .view-grouping-content h3 time {
+	//border-bottom: 1px solid $green;
+	display: block;
+	color: $green;
+	width: 100%;
+	margin-top: 1em;
+	margin-bottom: -0.30em;
+	border-bottom: 4px solid $green;
+}
+
+.program-schedule .view-grouping .view-grouping-content h3 {
+	font-size: 1.5em !important;
+}
+
+.program-schedule .views-row {
+	background: rgba($green, 0.2);
+	border-top: 2px solid $green;
+	padding: 0.5em 0.25em 1em 0.25em;
+}
+
+.program-schedule .views-row:nth-child(2) {
+	background: rgba(255,0,0,1);
+}
+
+
+.program-schedule .views-row:nth-child(even) {
+	background: rgba($green, 0.1);
+}
```

Comment:
@lkopacz I will take it out. As is often the case with components within View displays, there are a number of elements that need to be worked around.